### ISSUE TITLE
Sensor error: SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by imbaimbaimba** [custom-components/sensor.plex_recently_added#14]
**Describe the Bug**
I cannot get the sensor working. Home Assistant gives me the following error:
  File "/home/homeassistant/.homeassistant/custom_components/sensor/plex_recently_added.py", line 7
    <!DOCTYPE html>
    ^
SyntaxError: invalid syntax

**Indicate if you followed all steps in the [troubleshooting doc](https://github.com/custom-cards/upcoming-media-card/blob/master/troubleshooting.md)**
Well, did you? I tried.

**What data does the sensor display if any?**
Nothing

**What displays when visiting the API URL from the troubleshooting guide?**
It shows me the whole XML as intended

**Does the default Home Assistant Plex component work for you?**
Yes

**Home Assistant Version**
0.87.1

**Comment by maykar** [comment]
When you copied the code from github, did you copy from the raw file?

**Comment by maykar** [comment]
![image](https://user-images.githubusercontent.com/25127328/52600157-b7478d00-2e28-11e9-9974-8afa30e9eaaa.png)

**Comment by imbaimbaimba** [comment]
@maykar - No, should I? I just right-clicked and save-as on the file.

**Comment by imbaimbaimba** [comment]
@maykar - I just replaced it with right clicking Raw and saving as. Now it works!
Also the kodi sensor also started acting differently after I did the same to that one. It had the same error before, but now the kodi sensor gives me this:
TransportError trying to run API method None.VideoLibrary.GetRecentlyAddedEpisodes({'properties': ['art', 'dateadded', 'episode', 'fanart', 'firstaired', 'playcount', 'rating', 'runtime', 'season', 'showtitle', 'title']})

  File "/home/homeassistant/.homeassistant/custom_components/sensor/kodi_recently_added.py", line 100, in async_update
    self.data = result['episodes']
TypeError: 'NoneType' object is not subscriptable

Have any idea what that is?
